tests/controls/choicetest.cpp

```diff
@@ -89,4 +91,30 @@ void ChoiceTestCase::Sort()
 #endif
 }
 
+void ChoiceTestCase::GetBestSize()
+{
+    wxArrayString testitems;
+    testitems.Add("1");
+    testitems.Add("11");
+    m_choice->Append(testitems);
+
+    for ( int test = 0; test < 2; ++test )
+    {
+        if ( test == 1 )
+        {
+            // Ensure that the hidden contol return a valid best size too.
+            m_choice->Hide();
+        }
```

Comment:
This is not nice, we should be able to use `SECTION()` like this instead (warning: untested code ahead):

```cpp
void ChoiceTestCase::GetBestSize()
{
    m_choice->Append("1");
    m_choice->Append("11");

    SECTION("Normal best size")
    {
         // nothing to do here
    }

    SECTION("Hidden best size")
    {
         m_choice->Hide();
    }

    wxYield();

    m_choice->InvalidateBestSize();
    const wxSize bestSize = m_choice->GetBestSize();

    CHECK( ... );
}
```